Utilities/macrecovery/macrecovery.py

```diff
@@ -24,6 +24,7 @@
     from urlparse import urlparse
 
 SELF_DIR = os.path.dirname(os.path.realpath(__file__))
+COM_APPLE_RECOVERY_BOOT = 'com.apple.recovery.boot'
```

Comment:
`DOWNLOAD_DIR` may be a more appropriate variable name.

Reply:
Alright, done